lib/components/Card.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Button } from '@components/Button';
+
+const Card = () => {
+  return (
+    <div className="w-full border rounded-3xl border-stone-300">
+      <div className="grid grid-rows-[75px_minmax(0,_1fr)_100px] border-stone-300">
+        <div className="flex flex-col h-full border-b border-stone-300  justify-center">
+          <h1 className="text-xl font-medium text-left pl-4">Brukerinfo</h1>
+        </div>
+        <div className="grid grid-cols-2 border-b border-stone-300">
+          <div className="grid grid-rows-3 border-r border-stone-300">
+            <div className="border-b border-stone-300 text-left pl-4 py-3">
```

Comment:
Har brukt grid, vet ikke om det er den beste måten? Den skalerer ikke bra etter skjermstørrelse. Noen tips?

Reply:
Oooh, koden ser nice ut! Fancy toggle button jo haha. Visste ikke at man kunne ::after i tailwind.
- **Linje 7**: Kan gjøres om til flex, blir mer clean mtp. at vi får brukt tailwind spacing for høydene. `flex flex-col` og (`h-48` `h-full` `h-24`) på children elementene f.eks.

Du skalerer etter skjermstørrelse gjennom bruk av media queries på grid-cols: https://tailwindcss.com/docs/responsive-design.
- **Linje 11**: Legg til `md:grid-cols-1` f.eks.

Ellers ser ting veldig bra ut. Ser at figmadesignet har vært litt dårlig med margin tho, legg til litt på sidene kanskje :P